docs/libs.md

```diff
@@ -60,3 +60,6 @@ you'll need to first convert AsciiMath into LaTeX input, then call KaTeX.
 
 - [@rojer/katex-mini](https://github.com/rojer95/katex-mini): A Wechat Mini Program library that uses KaTeX for TeX math rendering.
```

Comment:
Accidental deletion? This should be kept, unless there's a reason to delete it.

Reply:
I just deleted the empty line at the bottom of the file, or should this also be kept?